src/fides/api/api/v1/endpoints/consent_settings_endpoints.py

```diff
@@ -0,0 +1,44 @@
+from fastapi import Depends
+from fastapi.params import Security
+from loguru import logger
+from sqlalchemy.orm import Session
+
+from fides.api.api import deps
+from fides.api.models.consent_settings import ConsentSettings
+from fides.api.oauth.utils import verify_oauth_client
+from fides.api.schemas.consent_settings import (
+    ConsentSettingsRequestSchema,
+    ConsentSettingsResponseSchema,
+)
+from fides.api.util.api_router import APIRouter
+from fides.common.api import scope_registry as scopes
+from fides.common.api.v1 import urn_registry as urls
+
+router = APIRouter(tags=["Consent Settings"], prefix=urls.V1_URL_PREFIX)
+
+
+@router.get(
+    urls.CONSENT_SETTINGS,
+    dependencies=[Security(verify_oauth_client, scopes=[scopes.CONSENT_SETTINGS_READ])],
+    response_model=ConsentSettingsResponseSchema,
+)
+def get_consent_settings(*, db: Session = Depends(deps.get_db)) -> ConsentSettings:
```

Comment:
a very specific comment (this is obviously one of the lest foundational pieces of this PR), but this caught my attention while watching your (very well done and informative!) loom: it may make sense to make this just any other "application setting" and [expose the setting](https://github.com/ethyca/fides/blob/main/src/fides/api/schemas/application_config.py) to our `[Config` API](https://github.com/ethyca/fides/blob/main/src/fides/api/api/v1/endpoints/config_endpoints.py#L41) so that it's settable via API?

i can discuss how that works if it's not clear from the code (code still needs to  be improved to make _all_ of our application settings configurable via API automatically, but that's been a longer-term TODO)!

anyway, it's not a big deal at all -- and i promise to go thru the more substantive pieces of this code soon! -- but figured i'd point it out quickly in case it's something you feel is appropriate to adjust!

Reply:
good callout, thank you Adam! I had forgotten about that work you did!

Reply:
ah right OK - that makes sense! seems to totally fine to keep it in its own table as you've got it then and keep things open for now. 

to be clear, the config API was more about making config properties settable via API (to allow updates without an app restart), rather than the reverse - so it's there's no real concern here!